### PR TITLE
chore(node): upgrade to version 16.13.0

### PR DESCRIPTION
**Pull request by thedaviddias**
***Short description of what this resolves:***

- Move from v12 to v16 for nvm
- Sorting of package.json using sort-package-json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -103,24 +123,8 @@
...
-      "src/**/*.ts",
-      "bin/htmlhint"
-    ],
-    "reporter": [
-      "text",
-      "lcov"
-    ],
-    "sourceMap": false
+  "engines": {
+    "node": ">=16.13.0 <= 16.99.99",
```
Comment (nschonni) [inline package.json:127]: I don't think this needs to get pinned to 16, or people will get warnings on install if they aren't using these versions.
Usually only need these to flag minimal versions of Node

Comment (thedaviddias) [reply package.json]: I removed the limit to a higher version.

File: package.json [hunk 2]
```diff
@@ -103,24 +123,8 @@
...
-      "src/**/*.ts",
-      "bin/htmlhint"
-    ],
-    "reporter": [
-      "text",
-      "lcov"
-    ],
-    "sourceMap": false
+  "engines": {
+    "node": ">=16.13.0",
```
Comment (nschonni) [inline package.json:127]: I'd probably drop both of these unless there is something needed in order for this to run once installed. I think there is a minimum version for stuff like ESM, but I think it's 12.20 something

Comment (thedaviddias) [reply package.json]: Yeah, actually, you're right, after thinking about it, probably better to remove this 🙏

File: package.json [hunk 3]
```diff
@@ -2,42 +2,46 @@
...
+    "build:rollup": "npm run build:min && npm run build:unmin",
     "build:unmin": "rollup -c",
+    "commit": "npx git-cz",
     "lint": "prettier -c . && eslint . --ext .js,.ts --max-warnings 0",
     "lint:fix": "npm run prettier && npm run lint -- --fix",
+    "prettier": "prettier --write .",
+    "test": "mocha --recursive \"./test/**/*.spec.js\"",
     "test:coverage": "nyc npm run test"
   },
   "husky": {
```
Comment (nschonni) [inline package.json:41]: sorry, I landed the Husky commit, so you might need to delete this node now

**Review by nschonni — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (thedaviddias) [reply package.json]: that's ok, we can keep these.

**Pull request merged**